README.md

```diff
@@ -93,6 +93,49 @@ Various exceptions can occur for malformed input files.
  * <treatment sample fold chages: fold changes 1..N> (postfixed _cf)
  * avgFC_cf: average fold change values based on corrected counts
 
+ List of plotly(html) and pdf plots generated for given analysis run
```

Comment:
(Inspired by my current task of figuring out the meaning and origin of files which have lost their metadata,) will these output files include any metadata about the inputs or flags?  Either machine readable or human readable.
I don't know whether it's worth the effort to add it...  it seems potentially useful but it's not part of this feature branch, and maybe not a requested feature at all.  Do you want a wishlist issue for it?